corehq/apps/users/templates/users/edit_commcare_user.html

```diff
@@ -117,6 +117,9 @@
 {% endblock %}
 
 {% block main_column %}
+    <p class="alert alert-warning">
+        {% trans "This is used is marked as demo user. All forms submitted by this user will be ignored." %}
```

Comment:
Should probably elaborate on 'ignored': `will be ignored and will not appear in reports or data exports.`